### PR TITLE
now works with Components that accept Observable as children

### PR DESCRIPTION
**Pull request by almasakchabayev**
Hi just wanted to submit PR related to this issue https://github.com/FormidableLabs/radium/issues/508

### CODE REVIEW

**Comment by ianobermiller** [comment]
This is probably a good start. We could also get fancy and wrap the subscribe to apply Radium to any elements emitted.

File: src/resolve-styles.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ const _resolveChildren = function({
 
   const childrenType = typeof children;
 
-  if (childrenType === 'string' || childrenType === 'number') {
+  if (childrenType === 'string' || childrenType === 'number' || children.subscribe) {
```
Comment (ianobermiller) [inline src/resolve-styles.js]: should we at least check that `subscribe` is a function?

**Comment by ianobermiller** [comment]
`subscribe` isn't _that_ universal, it seems that each FRP library uses different naming (bacon is `onValue`, for instance). But maybe this is good enough, since many people use RxJS.

**Comment by alexlande** [comment]
This one seems stalled out, going to close it. We can revisit in the future.

**Pull request closed without merging**